packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -142,7 +145,13 @@ export const AddToCartModal = () => {
                                             </Box>
 
                                             <Box>
-                                                <Text fontWeight="700">{product.name}</Text>
+                                                <Heading
+                                                    as="h2"
+                                                    fontSize="16px"
+                                                    fontWeight="semibold"
```

Comment:
Closer styles to what we were showing with the p tag:
```suggestion
                                                    fontSize="md"
                                                    fontFamily="body"
                                                    fontWeight="700"
```